src/twisted/python/test/test_release.py

```diff
@@ -1400,7 +1412,7 @@ def test_remove(self):
         """
         reposDir = self.makeRepository(self.tmpDir)
         testFile = reposDir.child('some-file')
-        testFile.setContent("something")
```

Comment:
I think what @exarkun is saying is not that you can't call `FilePath.child` with a type called `str` on either Python version (indeed you can, it's quite intentional ambidextrous when it comes to text and bytes), but rather that `str` is an ambiguous 2/3 type, and any code passing _what it considers to be a `str`_ (i.e.: it doesn't know whether it's really talking about text or bytes, so it's using an ambiguous type) has not really been through the process of porting.

Generally, I agree with this sentiment, but I'm willing to be a bit more lax with `_release.py` because of the lack of compatibility contract; we can address each call site in a small clean-up PR after the module is ported.